client/src/interface/Booking.ts

```diff
@@ -1,11 +1,11 @@
 import { User } from './User';
 
 export type Status = 'pending' | 'accepted' | 'declined' | 'paid';
+
 export interface Booking {
   _id: string;
   start: Date;
   end: Date;
   status: Status;
-  sitter: User;
-  owner: User;
+  otherUser: User;
```

Comment:
I'm personally against this change but I'd like to hear your reasoning as well 👍🏼